### ISSUE TITLE
Fails to run

### ISSUE THREAD

**Issue by hjf** [sibiraj-s/action-eslint#5]
Hello. I configured this action for one of my repos, but I get the following error:

```
Run sibiraj-s/action-eslint@v1
/opt/hostedtoolcache/node/14.16.1/x64/bin/node /home/runner/work/tracker/tracker/node_modules/eslint/bin/eslint .eslintrc.js db.js logger.js modules/logger-transport-socketio.js modules/logger-transport-sqlite.js modules/pipeline.js modules/radio-controller.js modules/satscheduler.js modules/schedulerunner.js modules/telegram.js modules/tleupdater.js modules/tracker-controller.js server.js --ignore-path .gitignore --quiet
Value for 'ignore-path' of type 'path::String' required.
Error: The process '/opt/hostedtoolcache/node/14.16.1/x64/bin/node' failed with exit code 2
```

But --ignore-path is specified in the command line.

The action file I'm using is:

```yml
name: Lint

on:
  pull_request:
  push:
    branches:
      - master

jobs:
  eslint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '14'
      - run: npm ci # or yarn install
      - uses: sibiraj-s/action-eslint@v1
        with:
          eslintArgs: '--ignore-path .gitignore --quiet'
          extensions: 'js,jsx,ts,tsx'
          annotations: true
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by sibiraj-s** [comment]
what is the version of eslint you are using? will you be able to share a minimal repo to reproduce this?

**Comment by satyambnsal** [comment]
@sibiraj-s facing the same issue 😢

**Comment by sibiraj-s** [comment]
Can you try pinning the version to `v1.1.0` and check?

**Comment by sibiraj-s** [comment]
Found the issue. The issue is with applying eslintArgs will fix them shortly.

**Comment by sibiraj-s** [comment]
Fixed in v1.1.2. Let me know if that works.

Before

```
eslintArgs: '--ignore-path .gitignore --quiet'
```

Now.

```
eslintArgs: '--ignore-path=.gitignore --quiet'
```

Released as patch, since it never worked earlier.
